TelephoneCallExample/Properties/AssemblyInfo.cs

```diff
@@ -2,35 +2,18 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following 
+// General Information about an assembly is controlled through the following
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("TelephoneCallExample")]
-[assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("<Not Specified>")]
+[assembly: AssemblyCompany("Microsoft")]

```

Comment:
Yeah i belive all that AssemblyCompany("Microsoft") appeared after creating a new xprojects.  I saw that diff before pushing,  but to me it was not a big  issue having "<Not Specified>" versus "Microsoft" for sample project. Will revert that